server/sonar-server/src/main/java/org/sonar/server/computation/measure/Measure.java

```diff
@@ -28,7 +28,7 @@
 import static com.google.common.base.Preconditions.checkState;
 import static java.util.Objects.requireNonNull;
 
-public final class Measure {
+public class Measure {
```

Comment:
final should not be removed on this class. Its contracts is complexe enough that to be guaranteed, the possibility to subclass it must be disabled